app/views/customers/sessions/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="sigin-container">
+  <div class="row text-center">
+    <%= image_tag "sendaguy.png", width: '60%' %> 
```

Comment:
Actually it's better to use double quotes only where you need this for string concatenation. For example I'm trying to use double quotes in html attributes, like <div class="some-class">, but in ruby I use single quotes, like `= image_tag 'sendaguy.png', width: '60%', concatenated: "##{dom_id(object)}"`